src/kixi/datastore/metadatastore/dynamodb.clj

```diff
@@ -36,7 +36,8 @@
   (str group-id "_" activity))
 
 (def activity-table-projection
-  #{::md/id ::md/provenance ::md/schema ::md/file-type ::md/name ::md/description})
+  #{::md/id ::md/provenance ::md/schema ::md/file-type ::md/name ::md/description
```

Comment:
The addition below means that the aggregate is no longer accurate: all new records will have the additional columns populated but pre-existing rows won't.  We can either live with this (yuck implications for search etc) or fix the pre-existing entries.  To fix we'd either have to hack from it by reading the info from other tables OR use (yet to be written) event-replay transformation tooling to add the additional data back in.

note: it's weird description doesn't seem to be in the table.